mkdocs.yml

```diff
@@ -168,6 +168,7 @@ pages:
             - 'How to contribute': 'community_resources/contributing.md'
             - 'Report and follow issues': 'community_resources/report_follow_issues.md'
             - 'Development guidelines': 'community_resources/development_guidelines.md'
+            - 'Contribute code': 'community_resources/code.md'
```

Comment:
I was thinking it might fit into `How to contribute`

But maybe we should then change headers so there is:
- How to contribute documentation
- How to contribute code

?

Reply:
I added another section to https://doc.ezplatform.com/en/latest/community_resources/contributing/. When it comes to the header I wrote it according to the convention here. @DominikaK

Reply:
Two arguments here: headings starting with "How to..." are harder to browse at a glance and they are a bit longish for the table of content, so I'm all for Konrad's/current version.

Reply:
Then `How to contribute'` => `Contribute doc` then on a few lines above, point is to be consistent and clear, how was not the point ;)

Reply:
ah, sorry, I see now that the How to contribute is a overview page, ignore my comments